secret-heroku.properties

```diff
@@ -0,0 +1,16 @@
+app.namespace=https://cs156-s21-team-5pm-1-courses.herokuapp.com/
```

Comment:
I wonder if we're supposed to have this file committed to main branch? I believe this would be exposing our API keys to the public or everyone who can access this repo.

Reply:
Yes that is correct. Even by pushing to this branch, these credentials are now public and we will have to reset them.

I'm going to assume that you added this file using `git add .` - this serves as a good example for why you should never do that unless you've done a `git status` beforehand and know exactly what you're adding.

Reply:
Oops yes, that is how it got on there. Yikes. Thanks for catching that Evan.

Reply:
How can I redo the pull request to not push those credentials?